cluster/common.sh

```diff
@@ -456,7 +468,7 @@ EOF
       exit 2
   esac
 
-  popd
+  popd "$@"
```

Comment:
this change seems suspect

Reply:
Good point, so I change to just suppress this warning, it seems ok. What do you think?